cmds/cpio/fs_unix.go

```diff
@@ -0,0 +1,49 @@
+// Copyright 2013-2017 the u-root Authors. All rights reserved
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+package main
+
+import (
+	"fmt"
+	"io"
+	"os"
+)
+
+func create(f *File) error {
+
+	m, err := cpioModetoMode(f.Mode)
+	if err != nil {
+		return err
+	}
+
+	switch m {
+	case os.ModeSocket:
```

Comment:
as for the binary blob, I'll look at the go tree again, but my impression is that they tend to prefer initialized arrays (like this one) to binary blobs.
thanks.